modules/git/repo_stats.go

```diff
@@ -95,22 +98,18 @@ func (repo *Repository) GetCodeActivityStats(fromTime time.Time, branch string)
 					continue
 				}
 				switch p {
-				case 1: // Separator
```

Comment:
Why where these comments deleted?

Reply:
Looks like while copy and paste the func the and then saved the file,  vscode linter removed or optimized these commands.